episodes/08-intro-to-geospatial-concepts.Rmd

```diff
@@ -6,155 +6,171 @@ author: 'Ana Petrović'
-| Celestial body | Earth | Earth |
-| Ellipsoid | WGS-84 | Bessel 1841 |
-| Prime meridian | International Reference Meridian | Greenwich |
+|------------------|---------------------------|---------------------------|
+| Definition     | Dynamic (relies on a datum which is not plate-fixed)     | Static (relies on a datum which is plate-fixed) |
+| Celestial body | Earth                                                    | Earth                                           |
+| Ellipsoid      | WGS-84                                                   | Bessel 1841                                     |
+| Prime meridian | International Reference Meridian                         | Greenwich                                       |
 | Datum          | World Geodetic System 1984 ensemble                      | Amersfoort                                      |
 | Projection     | Geographic (uses latitude and longitude for coordinates) | Projected (uses meters for coordinates)         |
 | Method         | Lat/long (Geodetic alias)                                | Oblique Stereographic Alternative               |
 | Units          | Degrees                                                  | Meters                                          |
 
-
 : Main properties of WGS 84 and Amersfoort / RD New coordinate reference systems
 
 The figure below shows the same city (Rotterdam) in these two CRS.
-    
-```{r fig.cap="Rotterdam in two different CRS", echo=FALSE}
+
+```{r fig-crs, fig.cap="Rotterdam in two different CRS", echo=FALSE}
 knitr::include_graphics("fig/rotterdam-crs.png")
 ```
 
-In addition to using different CRS, these two maps of Rotterdam also have different scales. 
-
+In addition to using different CRS, these two maps of Rotterdam also have different scales.
 
 ## Map scale
 
 **Map scale** measures the ratio between distance on a map and the corresponding distance on the ground. For example, on a 1:100 000 scale map, 1cm on the map equals 1km (100 000 cm) on the ground. Map scale can be expressed in the following three ways:
 
-|||
-|---|----------| 
-|Verbal: | 1 centimetre represents 250 meters |
-|Fraction: | 1:25000 |
-|Graphic: | ![](fig/scalebar.png){width=75%} |
+|           |                                    |
+|-----------|------------------------------------|
+| Verbal:   | 1 centimetre represents 250 meters |
+| Fraction: | 1:25000                            |
+| Graphic:  | ![](fig/scalebar.png){width="75%"} |
 
-::: challenge
+::: callout
+
+Note that the maps presented in this lesson do not use a scale bar. Instead, plot axes will serve that purpose.
+
+:::
+
+## Types of geospatial data
+
+The map of Rotterdam in the figure above shows the area of the city as a discrete feature with precise boundaries. This type of data is called vector. Vector data can have the form of points, lines and polygons (areas).
+
+```{r fig-vector, fig.cap="Same geographical area as raster and vector data (Source: ???)", echo=FALSE}
```

Comment:
```suggestion
```{r fig-vector, fig.cap="Same geographical area as raster and vector data", echo=FALSE}
```